src/published-components.js

```diff
@@ -41,6 +41,8 @@ const publishedComponents = {
   Search: "/src/components/Search/Search.jsx",
   TextField: "/src/components/TextField/TextField.jsx",
   Toast: "/src/components/Toast/Toast.jsx",
+  ToastButton: "/src/components/Toast/ToastButton/ToastButton.jsx",
+  ToastLink: "/src/components/Toast/ToastLink/ToastLink.jsx",
```

Comment:
are you sure we need this components available outside Toast component?

Reply:
yes, because the developer need to create the button/or link and pass them to the toast (similar to alert banner)